oceannavigator/frontend/src/components/OceanNavigator.jsx

```diff
@@ -272,16 +279,22 @@ export default class OceanNavigator extends React.Component {
 
 
       let newTime = 0;
+      let newStarttime = 0;
       const time_promise = this.get_timestamp_promise(dataset, newVariable);
       $.when(time_promise).done(function(time) {
         newTime = time[time.length-1].id;
-
+        // check the length of the timestamp array and set the start time according to that
+        if (time.length > 24)
+          newStarttime = time[time.length-25].id;
+        else
+          newStarttime = time[0].id;
```

Comment:
Could we move this bit of logic into its own function? Maybe something like `getStartTime`...The calculation looks fine to me though!